src/admin/components/JsonData/index.js

```diff
@@ -5,7 +5,7 @@ import './style.scss';
 
 export default class JsonData extends React.Component {
   static propTypes = {
-    data: PropTypes.object.isRequired,
+    data: PropTypes.shape({}).isRequired,
```

Comment:
I agree, it feels like changing to shape with an empty object is sort of sweeping the issue eslint is detecting under the carpet and if we're going to do that, we might as well relax this rule.

One thing I did find playing around with this is that the detection of shape props is a bit limited so as soon as you start using it you'll find you need to relax one of the unused props rules (to ignore shape props) because the current rules fail to detect the usage of the props as soon as you alias the object (which is a common pattern with destructuring).

Overall, I like the idea of a warning when someone uses `object` as its good to get into the practice of properly specifying the object shape. However, doing that is only really useful if using shape will provide warnings if the props aren't used or don't match up. What I'm not sure of is what the console will tell us via react's dev warnings vs these lint warnings.